Formula/o/openjazz.rb

```diff
@@ -1,10 +1,9 @@
 class Openjazz < Formula
   desc "Open source Jazz Jackrabit engine"
   homepage "http://www.alister.eu/jazz/oj/"
-  url "https://github.com/AlisterT/openjazz/archive/refs/tags/20190106.tar.gz"
-  sha256 "27da3ab32cb6b806502a213c435e1b3b6ecebb9f099592f71caf6574135b1662"
-  license "GPL-2.0"
-  revision 1
+  url "https://github.com/AlisterT/openjazz/archive/refs/tags/20231028.tar.gz"
+  sha256 "c45ff414dc846563ad7ae4b6c848f938ab695eb4ae6f958856b3fa409da0b8ac"
+  license "GPL-2.0-or-later"
```

Comment:
where did you find `or-later`? I did not find that.

Reply:
In https://github.com/AlisterT/openjazz/blob/master/licenses.txt.  It has:

```
This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.
```

Reply:
Yes, that is the default text for the license. It is then up to the author to pick a version in the file comments.

Reply:
Looks like https://github.com/AlisterT/openjazz/issues/43 has been open for five years to resolve that, and the original code author still hasn't replied.  I just updated to `-only` just in case.